package.json

```diff
@@ -43,6 +43,114 @@
     "test": "vscode-test"
   },
   "contributes": {
+    "walkthroughs": [
+      {
+        "id": "ocaml-onboarding",
+        "title": "OCaml Onboarding",
+        "description": "Learn how to install OCaml, set up opam, and configure essential tools for development.",
+        "steps": [
+          {
+            "id": "install-opam-windows",
+            "when": "isWindows",
+            "title": "Install opam",
+            "description": "opam is the package manager for OCaml. Installing opam also installs the OCaml compiler. On Windows 10 or 11, install it using Winget:\n\nRun `winget install Git.Git OCaml.opam` in PowerShell.\n\n[Install opam](command:ocaml.install-opam)",
+            "media": {
+              "image": "assets/opam_install.jpg",
+              "altText": "Information about some prompts the user may recieve while installing opam"
+            },
+            "completionEvents": [
+              "onCommand:ocaml.install-opam"
+            ]
+          },
+          {
+            "id": "install-opam-mac",
+            "when": "isMac",
+            "title": "Install opam",
+            "description": "opam is the package manager for OCaml. Installing opam also installs the OCaml compiler. Install it opam's official script installer.\n\n[Install opam](command:ocaml.install-opam)",
```

Comment:
The last phrase is missing some words. Also we should show the install command about to be runned and make the button more explicit. Same for the others.